docs/topics/rn-new-features.adoc

```diff
@@ -7,21 +7,46 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.MTA plugin for IntelliJ IDEA
+.Rules for migrating from Java EE8 to Jakarta EE9
+{ProductShortName} now includes a comprehensive set of rules to support the Java EE8 to Jakarta EE9 migration path by:
 
-The current release supports the {ProductName} plugin for IntelliJ IDEA. The plugin provides assistance directly in IntelliJ IDEA for developers making changes for a migration or modernization effort.
+* Replacing `javax` artifact dependencies declared within `pom.xml` files
+* Replacing `javax` package import statements
+* Renaming `javax` XML schema namespaces
+* Renaming `javax` prefixed properties
+* Renaming bootstrapping files
 
-The plugin is available through the link:https://plugins.jetbrains.com[IntelliJ marketplace].
+Nearly all these new rules have quick fixes so the changes that they support can be automated when using an {ProductShortName} plugin or extension within an IDE (Red Hat CodeReady Studio, Eclipse, Eclipse Che, IntelliJ or Microsoft Visual Studio Code).
```

Comment:
Not sure you need "Nearly all these new rules". I think you could say "Most rules contain quick fixes to automate source code changes when using an {MTA} plugin or extension.

You don't need "within IDE + list" because the plugin/extension only works in an IDE.